test/03_USM.test.js

```diff
@@ -93,18 +115,14 @@ contract('USM', (accounts) => {
 
   function shouldEqualApprox(x, y) {
     // Check that abs(x - y) < 0.0000001(x + y):
-    const diff = (x.gt(y) ? x.sub(y) : y.sub(x))
+    const diff = x.gt(y) ? x.sub(y) : y.sub(x)
     diff.should.be.bignumber.lt(x.add(y).div(new BN(1000000)))
   }
 
-  function fl(w) { // Converts a WAD fixed-point (eg, 2.7e18) to an unscaled float (2.7).  Of course may lose a bit of precision
-    return parseFloat(w) / 10**18
-  }
-
```

Comment:
We should hang onto this, I often use it in my `console.log()`s when debugging.

Reply:
yeah that makes sense, we could move it into its own `debugUtils` or something file. Aside from the reasons quoted above, when first getting into the tests it was hard to figure out what is used for testing and what isn't used or has some other purpose like this one